ship/src/main/scala/ch/epfl/bluebrain/nexus/ship/files/FileProcessor.scala

```diff
@@ -72,6 +74,14 @@ class FileProcessor private (
     {
       case a: ResourceAlreadyExists => logger.warn(a)("The resource already exists").as(ImportStatus.Dropped)
       case i: IncorrectRev          => logger.warn(i)("An incorrect revision has been provided").as(ImportStatus.Dropped)
+      case n: NoSuchKeyException    =>
```

Comment:
The file copier could handle the error itself (but it is not very important)

Reply:
I will change it a bit later